components/raftstore/src/store/snap.rs

```diff
@@ -1376,6 +1385,7 @@ struct SnapManagerCore {
     encryption_key_manager: Option<Arc<DataKeyManager>>,
     max_per_file_size: Arc<AtomicU64>,
     enable_multi_snapshot_files: Arc<AtomicBool>,
+    stats: Arc<RwLock<Vec<SnapshotStat>>>,
```

Comment:
How about utilize histogram to store the distribution of durations ? If it's precise enough. Then we can reduce the usage of memory and network.

Reply:
The stats will report pd, pd will depence on this to adjust store limit.